ci/environment-test-minimal-deps.yml

```diff
@@ -0,0 +1,73 @@
+name: bk-test
+channels:
+  - conda-forge
+dependencies:
+  - python=3.8
+
+  # runtime
+  - jinja2 >=2.7
+  - numpy >=1.11.3
+  - packaging >=16.8
+  - pillow >=4.0
+  - python-dateutil >=2.1
+  - pyyaml >=3.10
+  - tornado <=5.1
```

Comment:
@graingert perhaps this needs to be more strict e.g. `tornado 5.1*` ?

Reply:
I'm actually not sure on the syntax for conda environments